src/common/exceptions/http/not.found.http.exception.ts

```diff
@@ -0,0 +1,14 @@
+import { AlkemioErrorStatus, LogContext } from '@common/enums';
+import { BaseHttpException } from './base.http.exception';
+import { HttpStatus } from '@nestjs/common';
+
+export class NotFoundHttpException extends BaseHttpException {
+  constructor(message: string, context: LogContext, code?: AlkemioErrorStatus) {
+    super(
+      message,
+      HttpStatus.NOT_FOUND,
+      context,
+      code ?? AlkemioErrorStatus.NOT_FOUND
```

Comment:
Why would you need to pass anything here and not just use `AlkemioErrorStatus.NOT_FOUnD` in all cases? I can't imagine a scenario where you'd need something else.